StringCalculator.sln

```diff
@@ -0,0 +1,28 @@
+﻿
```

Comment:
General comment around your commits (ignore where this comment is, I just use the SLN and proj files to give general comments), rather use commit messages of what was actually changed/added to the code, i.e. "added delimiter functionality", or "updated delimiter splitting to use custom delimiters", etc.